localization/do.xml

```diff
@@ -7,12 +7,16 @@
 		<language iso_code="es" />
 	</languages>
 	<taxes>
-		<tax id="1" name="ITBIS DO 16%" rate="16" />
-		<tax id="2" name="ITBIS DO 12%" rate="12" />
+		<tax id="1" name="ITBIS DO 18%" rate="18" />
+		<tax id="2" name="ITBIS DO 13%" rate="13" />
 		
-		<taxRulesGroup name="DO Standard Rate (16%)">
+		<taxRulesGroup name="DO Standard Rate (18%)">
 			<taxRule iso_code_country="do" id_tax="1" />
 		</taxRulesGroup>
+    
+    <taxRulesGroup name="DO Reduced Rate (13%)">
+			<taxRule iso_code_country="do" id_tax="2" />
+		</taxRulesGroup>
 		
 		<taxRulesGroup name="DO Reduced Rate (12%)">
```

Comment:
@LouiseBonnard This taxRuleGroup must be removed. The id_tax 2 is now 13%.

Reply:
Oops, my bad, thanks Franck!